docs/VALUES.md

```diff
@@ -0,0 +1,18 @@
+# Core Values
+Shared by a broader community of the Organic Maps project developers and users.
+
+- **Free and Open Source**
+<br>The complete codebase and associated resources are released under Free and Open Source Software licenses and development happens in an open and transparent way. Refer to the [policy](FOSS_POLICY.md) for details and minor exceptions.
+- **Privacy**
+<br>No personal information is collected by default. And we make a reasonable effort to protect it from third parties. Check the [Privacy Policy][PRIVACY_POLICY.md] for details.
+- **Community**
+<br>The project is based on a broad community of volunteer contributors be it developers or donors or users answering each others questions and promoting OM to friends. And we're proud to be a part of a bigger OpenStreetMap community.
+<br>The project endeavours to have little hierarchy in its governance and strives to conduct as much discussion in public as possible.
+- **Core qualities of user-facing Organic Maps apps:**
+  - Easy to use
+  - Free
```

Comment:
Does it conflict with a possibility to have paid features?
Maybe it should be rephrased?
@rtsisyk WDYT?

Reply:
Free as in beer is a bit too aggressive to be sustainable